### ISSUE TITLE
Original Firmware Fix

### ISSUE THREAD

**Issue by chaddyyyy** [tinfever/FU-Dyson-BMS#34]
if this is not allowed tinfever feel free to remove this,

some useful info here lads...
31 flashing red lights,

![20230914_234145](https://github.com/tinfever/FU-Dyson-BMS/assets/124208891/b5492472-0772-46de-b72d-d740d7c59e96)
hook up the BMS.. no need to wake the battery up, if anything let it sleep.. or leave it to complete its red light blinking if triggered, let it fall alseep, then connect,

![Screenshot_20230914_234533_Gallery](https://github.com/tinfever/FU-Dyson-BMS/assets/124208891/cc51d0eb-2968-40dd-ac6f-fc7a524f2604)
make sure program memory is unchecked "disabled" or you will brick the original firmware,

![Screenshot_20230914_234908_Gallery](https://github.com/tinfever/FU-Dyson-BMS/assets/124208891/5a87f6ff-fbb7-4304-8ff6-c485a3b93744)
click auto import hex + write

![20230914_235001](https://github.com/tinfever/FU-Dyson-BMS/assets/124208891/9adcfba8-b7e9-47e2-8c21-cf3117bed656)
and find the working HEX you need,

downloads here.. extract the zip 

[DC59 61462-0107.zip](https://github.com/tinfever/FU-Dyson-BMS/files/12614265/DC59.61462-0107.zip)
[SV03 188002 0104.zip](https://github.com/tinfever/FU-Dyson-BMS/files/12614266/SV03.188002.0104.zip)
[SV04 188002 0104.zip](https://github.com/tinfever/FU-Dyson-BMS/files/12614267/SV04.188002.0104.zip)
[SV10 180207-0104.zip](https://github.com/tinfever/FU-Dyson-BMS/files/12614268/SV10.180207-0104.zip)
[SV11 228499 0103.zip](https://github.com/tinfever/FU-Dyson-BMS/files/12614269/SV11.228499.0103.zip)

notes..
if you still get 31 red flashes there's a problem with your batteries and needs more investigating, ie swopping for better ones,

if it works but when you connect to the dyson you get 10 red flashes when pulling the trigger, The voltages on the battery cells are dipping too much in voltage, you will need to connect a 10uf 16v cap to D3 & R48 like this..
![262178231-cd7c2c4b-e368-4655-94c2-1c2a65b9e28a](https://github.com/tinfever/FU-Dyson-BMS/assets/124208891/3d64c171-8edb-4411-b08b-7ce6ba834844)

i have revived loads of boards back to life.. not guarantee it will work for you, but from experience.. the boards I was not able to revive also had problems with aftermarket firmware with 16 red flashing.. brown outs, battery acid damage to the board etc etc

just quickly built this and shes a beast,
1701mah to 1750mah top row
1950mah to 2010mah bottom row, all holding at 41.50v to 41.80v after sitting for few weeks
still kept getting the 10 red flashing light on trigger so had to mod it.. i get V8 vibes from it good suction and long battery life :)
![20230914_221153](https://github.com/tinfever/FU-Dyson-BMS/assets/124208891/e7cfdbc8-cc94-4b67-9751-ae7de350703c)

**Comment by Smtblackstone** [comment]
Cihazın LED ışıklarında herhangi bir farklılık varmı acaba ? Teşekkürler

**Comment by chaddyyyy** [comment]
> Is there any difference in the LED lights of the device? Thanks

no this is the original firmware

**Comment by chaddyyyy** [comment]
> Is there any difference in the LED lights of the device? Thanks

no this is the original firmware

**Comment by Smtblackstone** [comment]
SV 11 zip hatası veriyor . Arşiv hasarlı hatası .yardımcı olabilirmisin teşekkürler

**Comment by chaddyyyy** [comment]
> SV 11 zip hatası veriyor . Arşiv hasarlı hatası .yardımcı olabilirmisin teşekkürler
@Smtblackstone 

rename the ZIP to RAR, GitHub won't allow RAR so I renamed it to ZIP, this might be the problem, also uploaded an HH11 dump for the V7 if the SV11 does not work.

**Comment by Smtblackstone** [comment]
Teşekkürler @chaddyyyy

**Comment by turenabo** [comment]
many thanks, have you tried on V8? hope you see this comment and bring us a V8 original firmware.

**Comment by chaddyyyy** [comment]
@turenabo SV11 is for the V8 🙂

**Comment by turenabo** [comment]
hi, maybe I misunderstood what you meant, I checked your file, they are all several hundred bytes, Tinfever's hex file is 40k bytes, so let me try to understand, your way is to add something into locked BMS' EEROM then unlocked the battery? thanks.

**Comment by chaddyyyy** [comment]
@turenabo yes import hex to eeprom data, make sure program memory is not enabled, check the pictures 🙂

**Comment by v47stepanenko** [comment]
worked flawlessly for my SV10 180207. appreciate it!

**Comment by zzp8202** [comment]
Thanks for share, it's worked on SV10K, before reset, it flash red light 4 times.

**Comment by oktaycalhan** [comment]
Hello, my Dyson V8 battery's SV10 BMS sometimes works, sometimes it flashes a red light 10 times. Where do I need to add a capacitor?

**Comment by pavelv441** [comment]
Hello. Will the SV11 firmware work on the SV22? Thanks.

**Comment by Pr0metheus2** [comment]
> SV11

Isn't SV22 model V15?

**Comment by pavelv441** [comment]
> > SV11
> 
> Isn't SV22 model V15?

Yes

**Comment by Pr0metheus2** [comment]
Then you are not on right place. Check here https://github.com/vladislav1983/V11_Dyson_BMS

**Comment by pavelv441** [comment]
Thanks.

**Comment by erayteknik07** [comment]
v12 bms yazılım indiremiyorum
